### ISSUE TITLE
[🐛] Yarn Workspaces check does not support `workspaces.packages` style

### ISSUE THREAD

**Issue by mhassan1** [snyk/snyk#1251]
- `node -v`: `v12.18.0`
- `npm -v`: `6.14.4`
- `snyk -v`: `1.352.1`
- OS: OS X
- Command run: `snyk test --yarn-workspaces --detection-depth=3 --strict-out-of-sync=false`

### Expected behaviour
Yarn Workspaces should be detected

### Actual behaviour
Yarn Workspaces are not detected

### Steps to reproduce
1. Create a Yarn Workspaces `package.json` (notice we are using `workspaces.packages: []` instead of `workspaces: []`, which was added in `yarn@1.4.2`: [link](https://classic.yarnpkg.com/blog/2018/02/15/nohoist/))
```
{
  "name": "my-workspace",
  "private": true,
  "workspaces": {
    "packages": [ "packages/*" ]
  }
}
```
2. Create a `packages/my-package/package.json`:
```
{
  "name": "my-package",
  "version": "0.0.0",
  "dependencies": {
    "lodash": "3.0.0"
  }
}
```
3. Run `snyk test --yarn-workspaces --detection-depth=3`

And I love the new Yarn Workspaces feature!

**Comment by lili2311** [comment]
👋 Thank you so much for flagging this up, responded to you in the PR. Let me know what you would like to do.

**Comment by mhassan1** [comment]
I've opened a PR here: https://github.com/snyk/nodejs-lockfile-parser/pull/76

Once merged, I'll create a PR here that bumps the dependency.

**Comment by gitphill** [comment]
@mhassan1 CLI with new nodejs-lockfile-parser was released today in v1.359.0
